Makefile

```diff
@@ -274,7 +274,10 @@ secrets:
 	hack/secrets.sh $(kerberos_id) $(kubeconfig_path) $(bw_password_path)
 .PHONY: secrets
 
+serviceaccount-secret-rotation:
+	go get github.com/openshift/ci-tools/cmd/serviceaccount-secret-rotation-trigger
```

Comment:
Do we want to use the image?
https://github.com/openshift/release/blob/2c5aa9c51abb17f55d9d3d8629f57fa3d7110d28/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L410